core/src/main/java/org/jruby/runtime/MixedModeIRBlockBody.java

```diff
@@ -15,29 +17,24 @@
 import org.jruby.util.log.Logger;
 import org.jruby.util.log.LoggerFactory;
 
-import java.io.ByteArrayOutputStream;
-
 public class MixedModeIRBlockBody extends IRBlockBody implements Compilable<CompiledIRBlockBody> {
     private static final Logger LOG = LoggerFactory.getLogger(MixedModeIRBlockBody.class);
 
     protected boolean pushScope;
     protected boolean reuseParentScope;
     private boolean displayedCFG = false; // FIXME: Remove when we find nicer way of logging CFG
-    private volatile int callCount = 0;
     private InterpreterContext interpreterContext;
+    private volatile int callCount = 0;
```

Comment:
Same thoughts about volatility 🤔

Reply:
yy, do not think it needs to be volatile.
but it kind of was before for the MixedModel impls